app/templates/apps/details.html

```diff
@@ -95,9 +95,9 @@ <h2 class="heading-medium">App data sources</h2>
 <hr>
 
 <!-- only show if signed in user has admin rights to this app -->
-<p class="clearfix align-right">
-  <a class="button button-secondary js-confirm" href="/apps/delete/{{ app.id }}/{{ user_id }}">Delete app</a>
-</p>
+<form action="{{ url_for('apps.delete', {app_id: app.id}) }}" method="post" class="clearfix">
```

Comment:
I think you may also need the `inline-form` class here as well.

Reply:
Not actually needed since there is only one button here, so it's fine to be a right-floated block.

Reply:
I see 🙇 I didn't realise that, it was needed in my case as I had two buttons side-by-side (I added these classes in my PR so once this is merged buttons will look awesome 🌟 )